apps/ledger-live-mobile/.eslintrc.js

```diff
@@ -5,15 +5,22 @@ module.exports = {
     "airbnb",
     "prettier",
     "plugin:json/recommended",
+    "plugin:@typescript-eslint/recommended",
   ],
   settings: {
+    "import/parsers": {
+      "@typescript-eslint/parser": [".ts", ".tsx"],
+    },
     "import/resolver": {
       node: {
         extensions: [".js", ".android.js", ".ios.js", ".ts", ".tsx"],
       },
+      typescript: {
+        alwaysTryTypes: true,
+      },
     },
   },
-  plugins: ["prettier", "detox"],
+  plugins: ["prettier", "detox", "@typescript-eslint", "import"],
```

Comment:
This should not be needed when using the recommended rules config for `@typescript-eslint` and `import` plugins (and even prettier if we use the correct config):
- https://github.com/typescript-eslint/typescript-eslint/tree/main/packages/eslint-plugin#usage
- https://github.com/import-js/eslint-plugin-import#typescript
- https://github.com/prettier/eslint-plugin-prettier#recommended-configuration
```suggestion
  plugins: ["detox"],
```